packages/base/src/dialogs/symbology/tiff_layer/types/MultibandColor.tsx

```diff
@@ -126,57 +126,52 @@ const MultibandColor = ({
   return (
     <div className="jp-gis-layer-symbology-container">
       <div className="jp-gis-band-container">
-        {loading ? (
-          <Spinner loading={loading} />
-        ) : (
```

Comment:
This check was superfluous; the overlay renders semi-transparently on top; and if it receives `props.loading = False`, it renders with `display: none`.